### PR TITLE
Validate antrea.yml file

### PR DESCRIPTION
**Pull request by ksamoray**
Perform a daily check to verify that antrea.yml file is up to date with
the file of antrea-io/antrea.

Signed-off-by: Kobi Samoray <ksamoray@vmware.com>

### CODE REVIEW

File: .github/workflows/validate_imports.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Validate antrea-manifest/antrea.yml
+        run: |
+          TMP_MANIFEST=$(mktemp /tmp/antrea.yml.XXXXXXX)
+
+          ./hack/generate-manifest.sh > $TMP_MANIFEST
+          cmp -s $TMP_MANIFEST ./antrea-manifest/antrea.yml
+          RET_CODE=$?
+          if [ $RET_CODE -ne 0 ]; then
+            echo "antrea.yml has changed in Antrea repository, update is required"
+            echo "Regenerate antera.yml using 'make antrea-manifest'"
```
Comment (salv-orlando) [inline .github/workflows/validate_imports.yml:22]: typo: antera

**Review by salv-orlando — COMMENTED** [review]
Just a typo

File: .github/workflows/validate_imports.yml (hunk 1 above)
Comment (ksamoray) [reply .github/workflows/validate_imports.yml]: Indeed

**Comment by jwsui** [comment]
Do we consider check role.yaml as well?

**Comment by ksamoray** [comment]
> Do we consider check role.yaml as well?

@jwsui once PR #49 is merged we'll validate role.yaml as well. Maybe in a separate PR.

**Comment by jwsui** [comment]
> > Do we consider check role.yaml as well?
> 
> @jwsui once PR #49 is merged we'll validate role.yaml as well. Maybe in a separate PR.

ok, thanks for the patch, LGTM.

**Review by salv-orlando — APPROVED** [review]
LGTM

**Pull request merged**